reporting-app/app/views/dashboard/_activity_report_approved.html.erb

```diff
@@ -1,4 +1,66 @@
+<%# Blue Banner Section %>
+<section class="usa-section usa-section--light padding-y-3 margin-bottom-4" aria-label="Get started">
+  <div class="grid-container">
+    <div class="grid-row flex-column tablet:flex-row gap-4">
+      <div class="usa-hero__callout grid-col">
+        <h2 class="usa-hero__heading font-heading-xl">
+          <span class="usa-hero__heading--alt"><%= t("dashboard.new_certification.get_started.header") %></span> <%= @certification.certification_requirements.due_date&.strftime('%b %d, %Y') %>
+        </h2>
+        <p class="text-base-lightest">
+         <%= t("dashboard.new_certification.get_started.prompt") %>
+        </p>
+        <%= link_to t("dashboard.new_certification.get_started.button"), exemption_screener_path(certification_case_id: @certification_case&.id), class: "usa-button" %>
+      </div>
+
+      <div class="grid-col display-flex flex-align-center">
+        <p class="font-serif-md">
+          <%= t("dashboard.new_certification.get_started.info") %>
+        </p>
+      </div>
+    </div>
+  </div>
+</section>
+
 <div class="grid-container">
-  <p class="usa-intro"><%= t(".intro") %></p>
-  <%= link_to t(".view_completed_certification_button"), certification_path(certification), class: "usa-button" %>
+  <%# Current Period Compliance Status Section %>
```

Comment:
Could this be extracted to a partial shared with _new_certification?

Reply:
yes I like that